packages/kokkos/core/src/impl/Kokkos_ChaseLev.hpp

```diff
@@ -190,7 +190,8 @@ struct ChaseLevDeque {
       return_value = *static_cast<T*>(a[b]);  // relaxed load
       if (t == b) {
         /* single last element in the queue. */
-#ifdef _WIN32
+// #ifdef _WIN32
+#if 0 // clang-cl says atomic_compare_exchange_strong is ambiguous.
```

Comment:
Again we are testing with visual studio during PR testing. I suppose we might not test clang-cl but the non-clang version of VS compiler but that one is a compiler we support. Does this change break that?